pills/06-our-first-derivation.xml

```diff
@@ -401,7 +401,7 @@
     coreutils is already in our nix store, no build is needed, it's already
     there with out path
     <filename>/nix/store/8w4cbiy7wqvaqsnsnb3zvabq1cp2zhyz-<emphasis
-    role="strong">coreutils-8.21</emphasis></filename>.
+    role="strong">coreutils-8.x</emphasis></filename>.
```

Comment:
I'd rather see a valid old path than an invalid one. The current path is at `/nix/store/dm20hrdk6s4jzfmk1197p2nya0p8fy3a-coreutils-8.29` and we will never see a path that ends with `8.x`.

Reply:
Fair enough, I updated it to the current valid path - of course that won't stay in sync

Reply:
That's true, but I think the reader will understand that it's an example after all; This is also true if the reader is using a different OS than the author.